portal-web/test/functional/com/liferay/portalweb/functions/AssertClick.function

```diff
@@ -97,6 +97,18 @@
 		<execute argument1="1000" selenium="pause" />
 	</command>
 
+	<command name="assertTextClickAtNoJavaScript">
+		<execute function="WaitForSPARefresh" />
+
+		<execute function="AssertTextEquals#assertText" />
+
+		<execute selenium="mouseOver" />
+
+		<execute selenium="clickAt" />
+
+		<execute selenium="assertLiferayErrors" />
```

Comment:
Hey @brianwulbern, I had a quick question - it seems that the only difference between this command and `AssertClickNoError` is the use of `assertLiferayErrors`. Do you think we still need to assert Liferay errors as we're clicking? Or are we okay with clicking and allowing the bookmark navigation to be our validation?

cc @austinchiang

Reply:
Hi @anthony-chu, yeah, I didn't want to gut the function any more than was necessary.  So I _removed_ `assertJavaScriptErrors` to make this new function, and kept `assertLiferayErrors` to err on the side of caution.

If we remove Liferay error assert here then no Bookmark test (whichclicks on a Bookmark link) will assert the console at this point.  I didn't want to do that, especially since it seems the problem is with the near-simultaneous-use of `getLocation` happening in both `assertJavaScriptErrors` and the immediately following assertLocation function.
```
	<command name="viewBookmark">
		<var name="key_assetName" value="${bookmarkName}" />

		<execute function="AssertTextEquals#assertPartialText" locator1="Bookmarks#DESCRIPTIVE_ENTRY_TITLE" value1="${bookmarkName}" />

		<execute function="AssertClick#assertTextClickAtNoJavaScript" locator1="Bookmarks#DESCRIPTIVE_ENTRY_TITLE" value1="${bookmarkName}" />

		<execute function="AssertLocation" value1="${bookmarkURL}/" />
	</command>
```
cc @austinchiang

Reply:
@anthony-chu rebased, see below :)